helm/managed/logging/elastic/values.yaml

```diff
@@ -21,6 +28,35 @@ elasticsearch:
       # Add default labels for the volumeClaimTemplate fo the StatefulSet
       #enabled: false
     #annotations: {}
+  extraEnvs:
+    - name: WAIT_ES_READY_IN_SEC
+      value: "100"
+    #- name: STATUS_COLOR
+    #  value: "yellow"
+  lifecycle:
+    postStart:
+      exec:
+        command:
+          - bash
+          - -c
+          - |
+            PIPELINE_NAME=hpccpipeline
+            ES_URL=http://localhost:9200
+            ES_CLUSTER_HEALTH_PATH=_cluster/health
+            ES_CLUSTER_HEALTH_PARAMS=_cluster/health
+            PIPELINE_REQ_PATH=_ingest/pipeline
+            [ -z ${WAIT_ES_READY_IN_SEC} ] && WAIT_ES_READY_IN_SEC=100
+            END_TIME=$(expr $(date +%s) \+ ${WAIT_ES_READY_IN_SEC})
+            ##START_FILE=/tmp/.es_start_file
+            ##ls $START_FILE
+            while [[ "$(curl -s -o /dev/null -w '%{http_code}\n' $ES_URL)" != "200" ]]; do
+              echo "\nES not ready..."
+              [ $(date +%s) -ge ${END_TIME} ] && break
+              sleep 1
+            done
+            #curl -X GET "$ES_URL/$ES_CLUSTER_HEALTH_PATH?wait_for_status=${STATUS_COLOR}&timeout=50s&pretty"  --fail --silent --show-error -o /dev/stderr
```

Comment:
why aren't we utilizing the built in health check with timeout?
Also, can $clusterHealthCheckParams be used here?

Reply:
from what I can see clusterHealthCheckParams is NOT being used in this curl command. I'm asking why you chose not to use it.

Beyond that, regarding your reply about "local=true"  what are the side affects of setting that value?